modules/lang-painless/src/main/java/org/elasticsearch/painless/node/ENewArrayFunctionRef.java

```diff
@@ -34,11 +35,11 @@
 /**
  * Represents a function reference.
  */
-public final class ENewArrayFunctionRef extends AExpression implements ILambda {
-    private final String type;
+public class ENewArrayFunctionRef extends AExpression implements ILambda {
 
-    private SFunction function;
-    private FunctionRef ref;
+    protected final String type;
+
+    // TODO: make local
```

Comment:
If this isn't gonna get fixed right away, consider cutting an issue to address this.

Reply:
Added issue.